lib/python-beta/tests/test_model.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
+
+from bailo import Client, Release
+from semantic_version import Version
+
+
+def test_release_version():
+    client = Client("https://example.com")
+
+    release = Release(client, 'test_id', '1.0.0', 1, "notes", [], [], draft=True)
```

Comment:
When there are this many parameters they should be named.  This is unintelligable!  Which of the last two arrays are images vs files, etc.